src/Reader/Reader.php

```diff
@@ -189,17 +197,14 @@ public static function useHttpConditionalGet($bool = true)
      */
     public static function import($uri, $etag = null, $lastModified = null)
     {
-        $cache       = self::getCache();
-        $client      = self::getHttpClient();
-        $client->resetParameters();
-        $headers = new ZendHttp\Headers();
-        $client->setHeaders($headers);
-        $client->setUri($uri);
+        $cache   = self::getCache();
```

Comment:
While I appreciate the comment, it's counter-productive to the intents of this proposal. Caching has been built-in to the component since the beginning, and removing that now would be a BC break. We can discuss doing so at a later date, for a major revision change.